src/Glpi/Controller/InventoryController.php

```diff
@@ -103,4 +103,38 @@ public function __invoke(Request $request): Response
         $response->setContent($inventory_request->getResponse());
         return $response;
     }
+
+    #[Route("/Inventory/RefusedEquipment", name: "glpi_refused_inventory", methods: 'POST')]
+    public function refused(Request $request): Response
```

Comment:
Just to have a method name that matches the corresponding URL chunk.
```suggestion
    public function refusedEquipement(Request $request): Response
```